app/values/validated_property.rb

```diff
@@ -0,0 +1,13 @@
+class ValidatedProperty
+  vattr_initialize :validated_property, :property_accessor
+end
+
+module Kernel
```

Comment:
The Kernel thing is a thought I had after I wrote it. It may be the next step is moving all this into lib and then doing the gem extraction, where I can have a clearly defined module to put things.